app/views/admin_shelters/index.html.erb

```diff
@@ -1,8 +1,15 @@
 <h1>All Shelters</h1>
 
-<% @shelters.each do |shelter| %>
+<% @shelters.reverse_ordered.each do |shelter| %>
   <div id="shelter-<%= shelter.id %>">
     <h3><%= shelter.name %></h3>
     <p>Created at: <%= shelter.created_at %></p>
   </div>
-<% end %>
\ No newline at end of file
+<% end %>
+
+<h3>Shelters with Pending Applications</h3>
+<section id="shelters_pending_applications"></p>
```

Comment:
The id tag is typically used with a unique identification like shelter.id. Did you intend to not pass an id? If so this should be a class tag and not an id tag

Reply:
id tag can be with anything in theory. Since the id "shelters_pending_applications" is unique on the page and the id won't be called again the use of 'id' is appropriate unless theres later functionality in different parts of the application.

Reply:
understand completely that id tag can be used in this case but it's not convention which is the only reason I mentioned it, https://www.w3docs.com/learn-css/css-id-and-class.html, 
if you don't want to change for convention's sake, I'll approve it